javascript/GridFieldExtensions.js

```diff
@@ -523,7 +523,7 @@
 				let currState = gridField.getState();
 				let toggleState = false;
 				let pjaxTarget = $(this).attr('data-pjax-target');
-				if ($(this).hasClass('font-icon-right-dir')) {
+				if ($(this).attr('aria-expanded') === 'false') {
```

Comment:
Ultimately the same functionality, except:
- the new condition is semantically correct ("if it's not expanded" instead of "if it has some css class")
- the icon class isn't on that element anymore so the alternative would be to find the child element first... which would be worse.